### PR TITLE
fix no default TableName of NewAdapterByDB()

### PR DESCRIPTION
**Pull request by miaolz123**
- style: format golang code
- feat: add default TableName of NewAdapterByDB()

### CODE REVIEW

**Comment by hsluoyz** [comment]
@miaolz123 please fix CI error.

**Review by 00LT00 — APPROVED** [review]
LGTM

File: adapter.go [hunk 1]
```diff
@@ -165,6 +165,10 @@ func NewAdapterByDBUseTableName(db *gorm.DB, prefix string,tablename string) (*A
 }
 
 func NewAdapterByDB(db *gorm.DB) (*Adapter, error) {
+	if db != nil && db.Statement.Table == "" {
+		db = db.Scopes(CasbinTableName(&CasbinRule{TableName: "casbin_rule"}))
```
Comment (00LT00) [inline adapter.go:169]: you must add `Session(&gorm.Session{WithConditions: true})` after the `db.Scopes()`

Comment (00LT00) [reply adapter.go:169]: refer to L250-L251

**Review by 00LT00 — CHANGES_REQUESTED** [review]
change that place can fix ci

**Review by 00LT00 — CHANGES_REQUESTED** [review]
please dont use `scopes` before all the SQL, it will bring a lot of duplicate code and doesn't help performance in any way.

**Comment by miaolz123** [comment]
OK, I will use `Session(&gorm.Session{WithConditions: true})` replace it,

**Comment by 00LT00** [comment]
I think you can update the func `NewAdapterByDBUserTableName` it has same problem. If you are interested, you can also add a test case to this function.

```go
func initAdapterWithGormInstanceByName(t *testing.T, db *gorm.DB, name string) *Adapter{
	//Create an Adapter
	a, _ := NewAdapterByDBUseTableName(db, "", name)
	// Initialize some policy in DB.
	initPolicy(t, a)
	// Now the DB has policy, so we can provide a normal use case.
	// Note: you don't need to look at the above code
	// if you already have a working DB with policy inside.

	return a
}
```

**Comment by miaolz123** [comment]
I have noticed this problem and I'll add more test cases soon.

**Pull request closed without merging**